src/wp-admin/theme-install.php

```diff
@@ -325,6 +325,10 @@
 					);
 				}
 				?>
+			<# } else if ( ! data.compatible_cp ) { #>
+				<?php
+				_e( 'FSE themes don\'t work with ClassicPress.' );
```

Comment:
Rather than escaping here maybe just change to double quotes:

`_e( "FSE themes don't work with ClassicPress." );`

Reply:
Fixed in [e744cf0](https://github.com/ClassicPress/ClassicPress-v2/pull/97/commits/e744cf02fb68ae0732e1367eb63ffb67626142ad).